modules/widgets/gallery.php

```diff
@@ -253,7 +253,7 @@ public function slideshow_widget( $args, $instance ) {
 
 		foreach ( $instance['attachments'] as $attachment ) {
 			$attachment_image_src = wp_get_attachment_image_src( $attachment->ID, 'full' );
-			$attachment_image_src = $attachment_image_src[0]; // [url, width, height]
+			$attachment_image_src = jetpack_photon_url( $attachment_image_src[0], array( 'w' => $this->_instance_width ) ); // [url, width, height]
```

Comment:
Should we have `$this->_instance_width` going through the `gallery_widget_content_width` filter, like it's done for the Gallery widget?